oxy/mapping.yaml

```diff
@@ -0,0 +1,54 @@
+!CANMsg
```

Comment:
I keep saying this lol but definitely would love to chat about how to best make this config sustainable

Reply:
Yeah this is kind of a mess. Not to feed the complexity demon, but we could have another layer of YAML - that is, humans write simple yaml -> translated into class-tagged yaml -> Rust